nvm.sh

```diff
@@ -2582,12 +2585,24 @@ nvm() {
             NVM_UPGRADE_NPM=1
             shift
           ;;
+          --default)
+            ALIAS='node'
+            shift
+          ;;
+          --alias=*)
+            ALIAS="${1##--alias=}"
+            shift
+          ;;
           *)
             break # stop parsing args
           ;;
         esac
       done
 
+      if [ -n "${ALIAS}" ]; then
+        nvm alias default "${ALIAS}"
```

Comment:
The intention was that `nvm install --alias=foo 8` would be like `nvm install 8 && nvm alias foo 8` - in other words, `--default` is a shortcut for `--alias=default`.
```suggestion
        nvm alias "${ALIAS}" $provided_version
```

Separately, this step shouldn't happen until after the installation is completed.